server/routes/referrals/referralsController.test.ts

```diff
@@ -50,10 +64,31 @@ describe('POST /referrals/start', () => {
   describe('when searching for a CRN found in Delius', () => {
     beforeEach(() => {
       communityApiService.getServiceUserByCRN.mockResolvedValue({
```

Comment:
Does the return value here matter if we're mocking out `serializeDeliusServiceUser` anyway? And could we use `deliusServiceUser` factory to create this object?

Reply:
You're right. I think it's quite nice to have the whole object in the test as a form of documentation, but I'll use the factory for it (I don't think it's got much use otherwise)